### PR TITLE
fix: add compatibility layer

### PR DESCRIPTION
**Pull request by menduz**
# What? 
This PR enables compatibility with normalized types in new versions of unity renderer

# Why? 
...

### CODE REVIEW

File: packages/unity-interface/BrowserInterface.ts [hunk 1]
```diff
@@ -379,13 +416,34 @@ export class BrowserInterface {
...
+        }
+      }
+      store.dispatch(saveProfileDelta(update))
     } else {
-      trackEvent('invalid_schema', { schema: 'SaveUserAvatar', payload: changes })
+      defaultLogger.error('error validating schema', validateRendererSaveProfileV1.errors)
+      trackEvent('invalid_schema', {
+        schema: 'SaveUserAvatar',
+        payload: changes,
+        errors: (validateRendererSaveProfileV1.errors ?? []).map(($) => $.message).join(',')
```
Comment (juanmahidalgo) [inline packages/unity-interface/BrowserInterface.ts:440]: Should we add logic here to know which `validate` fn failed? 
The `else` here will only  show the `validateRendererSaveProfileV1` errors even if `validateRendererSaveProfileV0` was the that failed.

**Review by juanmahidalgo — DISMISSED** [review]
Left a small comment but LGTM!

File: packages/unity-interface/BrowserInterface.ts (hunk 1 above)
Comment (menduz) [inline packages/unity-interface/BrowserInterface.ts:440]:
```suggestion
      defaultLogger.error('error validating schema', validateRendererSaveProfileV0.errors)
      trackEvent('invalid_schema', {
        schema: 'SaveUserAvatar',
        payload: changes,
        errors: (validateRendererSaveProfileV0.errors ?? []).map(($) => $.message).join(',')
```

File: packages/unity-interface/BrowserInterface.ts [hunk 2]
```diff
@@ -379,13 +416,34 @@ export class BrowserInterface {
...
+          snapshots: {
+            body: changes.body,
+            face256: changes.face256
+          }
+        }
+      }
+      store.dispatch(saveProfileDelta(update))
     } else {
-      trackEvent('invalid_schema', { schema: 'SaveUserAvatar', payload: changes })
+      defaultLogger.error('error validating schema', validateRendererSaveProfileV0.errors)
```
Comment (juanmahidalgo) [inline packages/unity-interface/BrowserInterface.ts:436]:
```suggestion
const errors = validateRendererSaveProfileV1.errors ?? validateRendererSaveProfileV0.errors
      defaultLogger.error('error validating schema', errors)
      trackEvent('invalid_schema', {
        schema: 'SaveUserAvatar',
        payload: changes,
        errors: (errors ?? []).map(($) => $.message).join(',')
      })
      defaultLogger.error('Unity sent invalid profile' + JSON.stringify(changes) + ' Errors: ' + JSON.stringify(errors))
```

Comment (juanmahidalgo) [reply packages/unity-interface/BrowserInterface.ts:436]: whoops, nvm, forgot to refresh and didn't see it merged.

**Pull request merged**